tasks/mina/rails.rb

```diff
@@ -19,6 +19,14 @@
   end
 end
 
+desc 'Starts an interactive database console'
+task :dbconsole do
+  set :execution_mode, :exec
+  in_path "#{fetch(:current_path)}" do
+    command %{#{fetch(:rails)} dbconsole}
+  end
+end
+
```

Comment:
can you rewrite it to use this template:
https://github.com/mina-deploy/mina/pull/672/files#diff-fe6c80d1b3934ea10fe1bc738791cda50ced69c723f18725fc87aba2077360ebR128

:)

Reply:
The link is referring to this PR, did you intend to link elsewhere?

Reply:
Yes i did, chechk the line it points to.
At the end of the file